### PR TITLE
Sign up form layout

### PR DESCRIPTION
**Pull request by Lucamuh**

### CODE REVIEW

File: source/package-lock.json [hunk 1]
```diff
@@ -3903,8 +3903,7 @@
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/boolbase/-/boolbase-1.0.0.tgz",
       "integrity": "sha1-aN/1++YMUes3cl6p4+0xDcwed24=",
-      "dev": true,
-      "optional": true
+      "dev": true
```
Comment (bbortt) [inline source/package-lock.json:3906]: which `npm` version are you using? mine is changing this back to `optional:true`. mine is:
```
$ npm version
{
  npm: '6.14.4',
  ares: '1.15.0',
  brotli: '1.0.7',
  cldr: '35.1',
  http_parser: '2.8.0',
  icu: '64.2',
  llhttp: '1.1.4',
  modules: '72',
  napi: '5',
  nghttp2: '1.39.2',
  node: '12.14.0',
  openssl: '1.1.1d',
  tz: '2019c',
  unicode: '12.1',
  uv: '1.33.1',
  v8: '7.7.299.13-node.16',
  zlib: '1.2.11'
}
```

File: source/src/main/webapp/styles/authentication/register.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+.register-info-text {
+  color: red;
+}
```
Comment (bbortt) [inline source/src/main/webapp/styles/authentication/register.css:4]: if you're using `register-info-test` only on I would recommend using `infoText.getCss().add("color", "red")`.

File: source/src/main/java/ch/bfh/bti7081/s2020/blue/view/authentication/RegistrationViewImpl.java [hunk 3]
```diff
@@ -15,9 +18,10 @@
 import com.vaadin.flow.router.PageTitle;
 import com.vaadin.flow.router.Route;
 
+@CssImport("./styles/authentication/register.css")
```
Comment (bbortt) [inline source/src/main/java/ch/bfh/bti7081/s2020/blue/view/authentication/RegistrationViewImpl.java:21]: you forgot to remove this annotation.. see [travis-ci build](https://travis-ci.com/github/zeroplexer/ch.bfh.bti7081.s2020.blue/jobs/336184252).

**Comment by Lucamuh** [comment]
removed css including part
reworked styling (removed nested vertical layout

**Pull request merged**